src/components/HTMLEngineProvider/BaseHTMLEngineProvider.js

```diff
@@ -231,6 +235,7 @@ const renderers = {
     code: CodeRenderer,
     img: ImgRenderer,
     edited: withLocalize(EditedRenderer),
+    'muted-text': TNodeChildrenRenderer,
```

Comment:
Just so I'm clear on why we are doing this, please correct me if I'm wrong...

From reading about it, I think a `TNodeChildrenRenderer` element is designed so it will render ONLY that element and its children when some data within the `TNodeChildrenRenderer` element changes. This prevents us from re-rendering a bunch of things unnecessarily. Is that correct?

Also, when we use it, why are we nesting it inside a `<RenderHTML` rather than a `<Text`?

Reply:
About `TNodeChildrenRenderer`... I actually didn't read the documentation about it, I just saw other usages like here:

https://github.com/Expensify/App/blob/e3e02c99a3c408af6222497e5a1b9327c7839c31/src/components/HTMLEngineProvider/BaseHTMLEngineProvider.js#L69-L111

I just needed something that rendered the rest of the html as it would by default, but allowed me to use the react native styles `mutedTextLabel`. I did the test of removing `'muted-text': TNodeChildrenRenderer,` and it works the same, seems like it is not doing anything useful.

The code 

```
'muted-text': defaultHTMLElementModels.p.extend({
    tagName: 'muted-text',
    mixedUAStyles: styles.mutedTextLabel,
}),
```
I would expect is would make the element `muted-text` render as a `<p>` with our `mutedTextLabel` styles, but in reality this is rendering it as a `<div>`. 

Using `defaultHTMLElementModels.div.extend` broke the layout, that's why I ended up using `p`, which for me made sense as a parent element of a message.

I'll do some testing on an Android device to confirm this is not working only in web.

>Also, when we use it, why are we nesting it inside a <RenderHTML rather than a <Text?

Because the `<Text` doesn't render html

Reply:
>  I did the test of removing 'muted-text': TNodeChildrenRenderer, and it works the same, seems like it is not doing anything useful.

Yeah...if it is not doing anything then lets get rid of it.